src/Microsoft.AspNetCore.Mvc.RazorPages/PageActionDescriptor.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.Abstractions;
+
+namespace Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure
+{
+    public class PageActionDescriptor : ActionDescriptor
```

Comment:
nit: It would be nice for this to have a friendly display name like ControllerActionDescriptor has